src/components/home/navbar/DropdownButton.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react'
+
+import DropdownMenu from './DropdownMenu'
+
+import '../../../styles/home/navbar/dropdown_button.css'
+
+class DropdownButton extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      isOpen: false
+    }
+    this.toggleMenu = this.toggleMenu.bind(this)
```

Comment:
Please bind inline, instead of in constructor (with whitespace between parens and params)